lib/json.dl

```diff
@@ -20,7 +20,7 @@ extern function to_json_string(x: 'T): Result<string, string>
 
 /* Represents any valid JSON value.
  */
-#[rust="serde(from = \"serde_json::value::Value\", into = \"serde_json::value::Value\")"]
+#[rust="serde(from = \"__json::ValueWrapper\", into = \"__json::ValueWrapper\")"]
```

Comment:
if the previous annotation was obscure, this one is doubly obscure.
Maybe once this API stabilizes we can wrap it into something that looks nicer for the user.

Reply:
Fortunately, this is internal to the library, users don't need to understand the annotations.